src/js/controller/model.js

```diff
@@ -288,7 +288,7 @@ define([
 
         // The model is also the mediaController for now
         this.loadVideo = function(item) {
-            this.mediaModel.set('playAttempt', true);
+            this.set('playAttempt', true);
```

Comment:
`mediaModel` would be better than `model` because it is reset when the item changes.